types/README.md

```diff
@@ -0,0 +1,11 @@
+# Updating protocols.json
+
+The main go-algorand repo contains a tool named `genconsensusconfig` that generates the supported consensus protocols version file.
```

Comment:
pretty sure the current protocols.json was generated with `goal protocols`

Reply:
It might have the advantage of using go-codec to already emit sorted keys, no need for jq. Seems to produce the same diff otherwise.